transaction-metrics-tracker/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+use {
+    lazy_static::lazy_static,
+    log::*,
+    rand::Rng,
+    solana_perf::sigverify::PacketError,
+    solana_sdk::{packet::Packet, short_vec::decode_shortu16_len, signature::SIGNATURE_BYTES},
+};
+
+// The mask is 12 bits long (1<<12 = 4096), it means the probability of matching
+// the transaction is 1/4096 assuming the portion being matched is random.
+lazy_static! {
+    static ref TXN_MASK: u16 = rand::thread_rng().gen_range(0..4096);
+}
+
+/// Check if a transaction given its signature matches the randomly selected mask.
+/// The signaure should be from the reference of Signature
+pub fn check_track_transaction(signature: &[u8; SIGNATURE_BYTES]) -> bool {
```

Comment:
```suggestion
pub fn should_track_transaction(signature: &[u8; SIGNATURE_BYTES]) -> bool {
```

"check" doesn't suggest what we're "checking" for, driving the developer to the definition to read comments and/or logic